documentation/index.html

```diff
@@ -13589,7 +13589,7 @@ <h2 id="style/taxi-edges">Taxi edges <a href="#style/taxi-edges"><span class="fa
 </ul>
 </li>
 <li><strong><code>taxi-turn</code></strong> : The distance along the primary axis where the first turn is applied.<ul>
-<li>This value may be an absolute distance (e.g. <code>20px</code>) or it may be a relative distance between the source and target (e.g. <code>50%</code>).</li>
+<li>This value may be an absolute distance - positive number (e.g. <code>20px</code>) to indicate distance from the source or negative number (e.g. <code>-20px</code>) to indicate distance from the target or it may be a relative distance between the source and target (e.g. <code>50%</code>).</li>
```

Comment:
Note that, in general, changes to the documentation should be made in their respective markdown files (and, when applicable, `docmaker.json`) as `index.html` is auto-generated from these. 

In this particular instance, the appropriate file to modify would be `documentation/md/style.md`

Reply:
Sorry. Totally missed it. Fixed